src/main/preload/mattermost.js

```diff
@@ -72,8 +72,6 @@ document.addEventListener('mouseup', () => {
 
 // listen for messages from the webapp
 window.addEventListener('message', ({origin, data: {type, message = {}} = {}} = {}) => {
-  console.log(`got a message from ${origin} of type ${type}`);
-  console.log(message);
   if (origin !== window.location.origin) {
     return;
```

Comment:
nit: here as well. Is it possible to do an autofix for these files once they are merged?